Makefile

```diff
@@ -14,7 +14,7 @@ init:
 	-docker rm -f \
 		mariadb postgres redis openldap \
 		mailserver_default mailserver_reverse mailserver_ecdsa mailserver_ldap mailserver_ldap2 \
-		mailserver_traefik_acmev1 mailserver_traefik_acmev2
+		mailserver_traefik_acmev1 mailserver_traefik_acmev2 || true
```

Comment:
What's going on here? It looks like we are trying to mask a failure on this line. What is it for?

Reply:
Yes, they are...
Since you may have a partial test running and some containers may be crashed or never been created.

So this is to ensure the cleanup is done with success.

Reply:
I noticed that when you don't have any running image, and you run the `docker rm` command with these options above, the command will fail because the images not found.  The `|| true` means if the command fails, we can continue the execution because I changed the command output to pass.  Also, in this line, we don't run any tests; this is just a cleanup line for docker.

Reply:
oh it's `rm`. Never mind me, did not notice that this is the removal part.